### ISSUE TITLE
really need feature that switch to the monited target window when click the replica window

### ISSUE THREAD

**Issue by kuailewangzi0328** [LorenzCK/OnTopReplica#131]
firstlly，thanks for developed this great amazing tool , it's realy helpful for me ~ 
I always use this to monitor a lot same game window, but when I find some information in one of replica window, I need  switch  to the target game window to handle it quickly. but now I can't switch to this game window by click this replica window directly, I must spend a lot time to find which game window is the correct window to handle it.
I want to add this feature with your code. but I have no write csharp code before. So can you tell me if this feature is feasible？ and how much workload if it is feasible~thanks very much

**Comment by LorenzCK** [comment]
Hi @kuailewangzi0328, thank you!
A feature like the one you describe should already exist: when you right-click on OnTopReplica and select “Switch to window”, this will hide the OnTopReplica window (it will be minimized) and bring the cloned window to foreground. Does this look like what you had in mind?

**Comment by kuailewangzi0328** [comment]
yes, but just a little different~ I hope it can bring the cloned window to forground immediately when I click the replica window, and do not minimized it self. 
because you know ,when I open many game window, I need sequential switch to every window  and  check if it is  ready oftentimes , so right click a replica window and select a option is too inconvenient for me.
so,I am very appreciated if you can help to add this "click-switch" mode in this app. thank again~
